scripts/planning/sprint-planning.sh

```diff
@@ -52,7 +52,7 @@ function github_milestone()
 
 function jira_milestone()
 {
-  milestone_data=`curl -s -X GET -H "Content-Type: application/json" "https://issues.jboss.org/rest/api/2/search?jql=project=CDK%20AND%20component=minishift%20AND%20fixVersion=${milestone}"`
+  milestone_data=`curl -s -X GET -H "Content-Type: application/json" "https://issues.jboss.org/rest/api/2/search?jql=project=CDK%20AND%20fixVersion=${milestone}"`
```

Comment:
@LalatenduMohanty I am not sure if I am doing something wrong but not able to run that curl command on my mac/Linux box

```
$ curl -s -X GET -H "Content-Type: application/json" "https://issues.jboss.org/rest/api/2/search?jql=project=CDK%20AND%20fixVersion=${milestone}"
{"errorMessages":["Error in JQL Query: Expecting either a value, list or function before the end of the query."],"errors":{}}
```

Reply:
nvm, I forgot to put milestone and now able to get the list.